sources/core/Stride.Core.Design/CodeEditor/CodeEditor.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Stride contributors (https://stride3d.net)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+
+namespace Stride.Core.CodeEditor
+{
+    public class CodeEditor
```

Comment:
``IDEInfo`` was alright, not sure why you changed that type's name ? ``CodeEditorInfo`` is fine too if you want to keep it that way.